roles/scilla/templates/systemd.service.j2

```diff
@@ -2,7 +2,7 @@
 Description={{ service_identifier }}
 
 [Service]
-User={{ service_identifier }}
+User=zilliqa
```

Comment:
Why do we need the same user ? Isnt the socket file is already 755 ?

Reply:
Also lets add those flags and move these two directories in /opt/scilla/socket and /opt/scilla/libs/extra

Reply:
> Why do we need the same user ? Isnt the socket file is already 755 ?

Lets change the group to project like we did in other repositories and see if that fixes it ?